src/function/FunctionGraph.ts

```diff
@@ -14,59 +13,11 @@ import {
-import {Locale} from "../config/Locale";
+import {drawGraphElement} from "./FunctionDraw";
 
 
-let mvp1IRI = "https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1";
-let mvp2IRI = "https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-2";
-
-export function drawGraphElement(elem: joint.dia.Element, languageCode: string, representation: number) {
-    if (typeof elem.id === "string") {
-        let types = VocabularyElements[ProjectElements[elem.id].iri].types;
-        let label = VocabularyElements[ProjectElements[elem.id].iri].labels[languageCode];
-        let labels: string[] = [];
-        if (ProjectSettings.viewStereotypes)
-            getStereotypeList(types, languageCode).forEach((str) => labels.push("«" + str.toLowerCase() + "»"));
-        labels.push(label === "" ? "<blank>" : label);
-        elem.prop('attrs/label/text', labels.join("\n"));
-        let text = [];
-        if (representation === Representation.COMPACT) {
-            for (let link in ProjectLinks) {
-                if ((ProjectLinks[link].source === elem.id || ProjectLinks[link].target === elem.id) &&
-                    ProjectLinks[link].active) {
-                    if (ProjectLinks[link].iri === parsePrefix("z-sgov-pojem", "má-vlastnost") &&
-                        ProjectLinks[link].source === elem.id && ProjectLinks[link].active) {
-                        text.push(VocabularyElements[ProjectElements[ProjectLinks[link].target].iri].labels[languageCode])
-                    } else if (ProjectLinks[link].iri === parsePrefix("z-sgov-pojem", "je-vlastností") &&
-                        ProjectLinks[link].target === elem.id && ProjectLinks[link].active) {
-                        text.push(VocabularyElements[ProjectElements[ProjectLinks[link].source].iri].labels[languageCode])
-                    }
-                }
-            }
-        }
-        elem.prop("attrs/labelAttrs/text", text.join("\n"));
-        let width = representation === Representation.COMPACT ?
-            Math.max((labels.reduce((a, b) => a.length > b.length ? a : b, "").length * 10) + 4,
-                text.length > 0 ? 8 * (text.reduce((a, b) => a.length > b.length ? a : b, "").length) : 0) :
-            (labels.reduce((a, b) => a.length > b.length ? a : b, "").length * 10) + 4;
-        elem.prop('attrs/text/x', width / 2);
-        let attrHeight = (24 + ((labels.length - 1) * 18));
-        let height = (text.length > 0 ? (4 + (text.length * 14)) : 0) +
-            attrHeight;
-        elem.prop('attrs/labelAttrs/y', attrHeight);
-        setElementShape(elem, width, height);
-        elem.resize(width, height);
-    }
-}
-
-export function getNewLink(type?: number, id?: string): joint.dia.Link {
-    let link = new joint.shapes.standard.Link({id: id});
-    if (type && type in LinkConfig) {
-        link = LinkConfig[type].newLink(id);
-    }
-    return link;
-}
+export var mvp1IRI = "https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1";
```

Comment:
vars are obsolete AFAIK, https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/